src/NBitcoin/ChainBase.cs

```diff
@@ -4,15 +4,33 @@
 
 namespace NBitcoin
 {
+    /// <summary>Base implementation for chains.</summary>
     public abstract class ChainBase
     {
+        /// <summary>
+        /// Gets the block given a block id(hash).
+        /// </summary>
+        /// <param name="id">Block id to retreive.</param>
+        /// <returns>The block.</returns>
         public abstract ChainedBlock GetBlock(uint256 id);
+
+        /// <summary>
+        /// Gets the block at a given block height.
+        /// </summary>
+        /// <param name="height">Height to retreive block at.</param>
```

Comment:
typo `ei`, i'd rephrase `Height of the block which header to retrieve.`

Reply:
Sounds good thanks.